### PR TITLE
feat: use buildpack

### PR DESCRIPTION
**Pull request by viceice**
ref: https://github.com/renovatebot/renovate/issues/5974

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,7 @@
-FROM renovate/ubuntu:18.04@sha256:f94cb492cb94e913186e477168dff87ab4c12c08a445d65d511c0cc628369d69
+FROM renovate/buildpack
 
-USER root
+ENV JAVA_VERSION=8
```
Comment (rarkins) [inline Dockerfile]: I had expected it to be `ARG`. Which takes precedence if it's also set in environment when run?

Comment (viceice) [reply Dockerfile]: When using `ARG` it will not exist on runtime, so i could do

```Dockerfile
ARG JAVA_VERSION=8
ENV JAVA_VERSION=${JAVA_VERSION}

Comment (rarkins) [reply Dockerfile]: I wasn't aware of that distinction. I just figured we may want the ability to supply the value from CLI

File: Dockerfile [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
-USER root
+ARG JAVA_VERSION=8
 
-RUN apt-get update && apt-get install -y default-jre-headless && \
-    rm -rf /var/lib/apt/lists/*
+LABEL maintainer="Rhys Arkins <rhys@arkins.net>"
+LABEL name="renovate-java"
+LABEL org.opencontainers.image.source="https://github.com/renovatebot/docker-java" \
+      org.opencontainers.image.url="https://renovatebot.com" \
+      org.opencontainers.image.licenses="MIT" \
```
Comment (rarkins) [inline Dockerfile]: Is this correct for openjdk though?

Comment (viceice) [reply Dockerfile]: 🤔 i thought it was for our image, but it's ment for the containing software, so i'll remove licenses

**Pull request merged**